src/pages/metapackages/index.md

```diff
@@ -4,7 +4,9 @@ title: Metapackages
 
 # Metapackages
 
-With PWA Studio 12.1.0, you can now use metapackages to extend Magento Open Source backends with new features and sample data. As an example, in release 12.1.0 we used the [Open Source metapackage][] to extend the GraphQL schema with a new mutation and a new field providing more information about cart item errors. These packages can be used to add similar backend functionality to your projects.
+PWA Studio uses metapackages to extend Magento Open Source backends with new features and sample data. For example, new GraphQL mutations and interfaces enable enhanced frontend capabilities.
+
+As of version 12.1.0, the GraphQL metapackage is required for PWA Studio.
```

Comment:
So is the "GraphQL metapackage" the agreed upon name by Dave and others? I'm not sure what the current status of all the metapackages is at the moment. I just know that it's been warmly contested.

Reply:
We were going to change it but we changed our minds last minute due to confusion about the scope of the extension. I missed reverting this instance. Fixed.